data/src/main/java/com/android254/data/di/NetworkModule.kt

```diff
@@ -43,8 +48,15 @@ object NetworkModule {
 
     @Provides
     @Singleton
-    fun provideHttpClientEngine(): HttpClientEngine = Android.create {
-        connectTimeout = 10_000
+    fun provideHttpClientEngine(@ApplicationContext context: Context): HttpClientEngine = OkHttp.create {
+        addInterceptor(
+            ChuckerInterceptor.Builder(context)
```

Comment:
Since we might add extra interceptors in the future, I would suggest you provide the `ChuckerInterceptor` via Hilt, like how we provide `provideHttpClient` or `provideTokenProvider`

Reply:
yeah, that's a good idea !